src/cabi.rs

```diff
@@ -176,6 +183,22 @@ export!(lsm_view_lookup_token(
     })
 });
 
+export!(lsm_view_get_original_function_name(
+        view: *const View, line: c_uint, col: c_uint, minified_name: *const c_char,
+        minified_source: *const c_char, name_out: *mut *const c_char) -> Result<c_uint>
+{
+    Ok(match (*view).get_original_function_name(
+        line as u32, col as u32, CStr::from_ptr(minified_name).to_str()?,
+        CStr::from_ptr(minified_source).to_str()?)
+    {
+        Some(name) => {
+            *name_out = name.as_ptr() as *const c_char;
```

Comment:
Would you care to gracefully return if parameters like `name_out` or even `minified_name` are null?